### DISCUSSION TITLE
Is this possible to run docker in post upgrade tasks ?

### DISCUSSION THREAD

**Discussion by eddycharly** [renovatebot/renovate#12318] [Request Help] [answered]
Hello, we are trying to setup a post upgrade task to update our terraform docs but it doesn't seem to work.

It is trying to run a docker image like this:
```json
  "postUpgradeTasks": {
    "commands": [
      "docker run --rm --volume ${PWD}:/terraform-docs quay.io/terraform-docs/terraform-docs:0.14.1 markdown /terraform-docs"
    ],
    "fileFilters": [
      "**/*.md"
    ],
    "executionMode": "update"
  }
```

Is this supposed to be supported ? Is there another recommended alternative ?

Thanks 🙏

**Comment by rarkins** [comment, marked as answer]
How are you running Renovate?

**Reply by eddycharly** [reply]
Not sure i understand the question 🤔
I think it's in hosted mode (Whitesource Renovate ?)

**Reply by rarkins** [reply]
Then no, you'd need to self host instead

**Reply by eddycharly** [reply]
Ok, thanks.
